docs/resources/nginx_module.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: The NginxModule Resource
```

Comment:
This should be `The nginx_module Resource` - use the resource DSL name instead of the Ruby class name.